src/lexer/mod.rs

```diff
@@ -400,7 +406,7 @@ impl Cursor<'_> {
                 '_' => {
                     self.bump();
                 }
-                '0'..='9' => {
+                '0'..='9' | 'a'..='f' | 'A'..='F' => {
```

Comment:
Bug?

```
let bruh = 12ffff
```

Reply:
The parser takes care of that:
https://github.com/antimony-lang/antimony/blob/e8b9c64cc4f623962bf47a4b99665a03c3967c8b/src/parser/rules.rs#L330-L349

Reply:
That's more of a hack, stuff like `11AA` as 10 radix integer shouldn't be allowed in the first place

Reply:
> Feel free to send a patch for this.

Alright, sent that to the mailing list + added tests for numbers